src/test/java/com/rakovets/course/java/core/practice/oop/principles/PersianTest.java

```diff
@@ -0,0 +1,53 @@
+package com.rakovets.course.java.core.practice.oop.principles;
+
+import com.rakovets.course.java.core.practice.oop.principles.Cat;
+import com.rakovets.course.java.core.practice.oop.principles.Siamese;
+import com.rakovets.course.java.core.practice.oop.principles.Sphynx;
+import com.rakovets.course.java.core.practice.oop.principles.Persian;
+import com.rakovets.course.java.core.practice.oop.principles.Person;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class PersianTest {
+
+    static Stream<Arguments> ProviderArgumentsForPersian() {
+        return Stream.of(
+                Arguments.of("Varya", "Varya MUURRR MEOW"),
+                Arguments.of("Tiger", "Tiger MUURRR MEOW"),
+                Arguments.of("Leon", "Leon MUURRR MEOW"),
+                Arguments.of("Murka", "Murka MUURRR MEOW")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("ProviderArgumentsForPersian")
+    void persianTest(String str1, String expected) {
+        Persian persian = new Persian(str1);
+        String actual = persian.getName() + " " + persian.catPurr() + " " + persian.catSayMew();
+        Assertions.assertEquals(expected, actual);
```

Comment:
"Violation: There isn't separating empty lines for GIVEN, WHEN, THEN parts
Solution: Add blank line"